### PR TITLE
Disable "NetworkProgrammingLatency" measurement in kubemark

### PR DESCRIPTION
**Pull request by oxddr**
/assign @wojtek-t

### CODE REVIEW

File: clusterloader2/pkg/measurement/common/slos/prometheus_measurement.go [hunk 1]
```diff
@@ -60,6 +61,10 @@ func (m *prometheusMeasurement) Execute(config *measurement.MeasurementConfig) (
 		return nil, nil
 	}
 
+	if !m.gatherer.IsEnabled(config) {
+		klog.Warningf("%s is disabled, skipping the measuerment", m)
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/common/slos/prometheus_measurement.go]: should be some return here :)

Comment (wojtek-t) [reply clusterloader2/pkg/measurement/common/slos/prometheus_measurement.go]: Also, for consistency maybe have the same log as here:
https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/pkg/measurement/common/probes/probes.go#L85

Comment (oxddr) [reply clusterloader2/pkg/measurement/common/slos/prometheus_measurement.go]: Added. 

Left the message as it was - this is general piece of code, not tied a specific measurement. I'd rather not add Kubemark specific message in here.

**Comment by oxddr** [comment]
PTAL

File: clusterloader2/pkg/measurement/common/slos/network_programming.go [hunk 2]
```diff
@@ -47,6 +47,14 @@ func init() {
 
 type netProgGatherer struct{}
 
+func (n *netProgGatherer) IsEnabled(config *measurement.MeasurementConfig) bool {
+	if config.CloudProvider == "kubemark" {
+		klog.Warningf("%s is disabled in Kubemark", n)
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/common/slos/network_programming.go]: I would remove this warning - logging twice that something is disabled doen't make sense to me.

File: clusterloader2/pkg/measurement/common/slos/network_programming.go [hunk 3]
```diff
@@ -47,6 +47,14 @@ func init() {
...
+		return false
+	}
+	return true
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/common/slos/network_programming.go]: in fact, this should just be

return config.CloudProvider != "kubemark"

**Comment by oxddr** [comment]
ref https://github.com/kubernetes/perf-tests/issues/615

**Pull request merged**